README.rst

```diff
@@ -25,13 +25,16 @@ Development
   make
 
 
-Create a tunnel to localhost using something like https://ngrok.com/download::
+If your sentry server is hosted behind a firewall without publich IP, you should
```

Comment:
I agree, but in some cases hipchat and sentry are in different LANs that
are difficult to connect, ngrok could make it easier. It's ok to remove
ngrok part from readme, but the situation should deserve to be considered.
Matt Robenolt <notifications@github.com>于2016年6月19日 周日下午10:34写道：

> In README.rst
> <https://github.com/getsentry/sentry-hipchat-ac/pull/28#discussion_r67617943>
> :
>
> > @@ -25,13 +25,16 @@ Development
> >    make
> >
> >
> > -Create a tunnel to localhost using something like https://ngrok.com/download::
> > +If your sentry server is hosted behind a firewall without publich IP, you should
>
> I'm not sure if I'd recommend using ngrok for any production environment.
> It's a developer tool.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/getsentry/sentry-hipchat-ac/pull/28/files/b71cfe20aae14a9bb11f8d21463687b9954962c8#r67617943>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAYx1Lw2IZF-RjYKKa06_1aT5zmbVTljks5qNVOJgaJpZM4I5KNi>
> .
>